src/modules/flight_mode_manager/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -51,22 +51,28 @@ bool FlightTaskOrbit::applyCommandParameters(const vehicle_command_s &command)
 {
 	bool ret = true;
 	// save previous velocity and roatation direction
-	float v = fabsf(_v);
-	bool clockwise = _v > 0;
+	bool is_clockwise = _orbit_velocity > 0;
+
+	float new_radius = _orbit_radius;
+	float new_abs_velocity = fabsf(_orbit_velocity);
 
 	// commanded radius
 	if (PX4_ISFINITE(command.param1)) {
-		clockwise = command.param1 > 0;
-		const float r = fabsf(command.param1);
-		ret = ret && setRadius(r);
+		// Note: Radius sign is defined as orbit direction in MAVLINK
+		float radius = fabsf(command.param1);
+		is_clockwise = radius > 0;
```

Comment:
```suggestion
		float radius = command.param1;
		is_clockwise = radius > 0;
```
Otherwise it's always > 0

Reply:
Fixed in bd289cc and verified